features/support/commit_helpers.rb

```diff
@@ -79,12 +84,11 @@ def committed_files sha
 #
 # rubocop:disable MethodLength
 def commits_for_branch branch_name
-  array_output_of("git log #{branch_name} --oneline").map do |commit|
+  array_output_of("git log #{branch_name} --oneline --topo-order --reverse").map do |commit|
```

Comment:
I like topo order since when we merge we list all the children from one side and then all the children from the other. Time didn't work when I tried it earlier (even did a date override to ensure that each commit had a timestamp one second later then the other)